Digipost/APIClient.swift

```diff
@@ -352,6 +330,7 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
             }, error: nil)
         urlRequest.setValue("*/*", forHTTPHeaderField: "Accept")
         fileTransferSessionManager.setTaskDidCompleteBlock { (session, task, error) -> Void in
+
```

Comment:
No, Alamofire calls both the closures. This will be fixed when removing alamofire